lib/Onyx.js

```diff
@@ -579,8 +570,15 @@ function init({
     keys,
     initialKeyStates,
     safeEvictionKeys,
-    registerStorageEventListener
+    registerStorageEventListener,
+    captureMetrics = false,
 }) {
+    if (captureMetrics) {
+        // The code here is only bundled and applied when the captureMetrics is set
+        // eslint-disable-next-line no-use-before-define
+        applyDecorators();
+    }
+
```

Comment:
The initial idea was to use `process.env.BENCHMARK_ONYX` ENV variable directly here, but it seems it's not as straightforward as to include it in `.env` or provide it form a shell

Variables in `.env` are exposed through `react-native-config` but since Onyx does not have it as dependency I've decided to use the `captureMetrics` flag

I'm settings `captureMetrics` in E.cash by reading env variables using `react-native-config` 

```js
import Config from 'react-native-config';
import Onyx, {withOnyx} from 'react-native-onyx';

Onyx.init({
    keys: ONYXKEYS,
    safeEvictionKeys: [ONYXKEYS.COLLECTION.REPORT_ACTIONS],
    captureMetrics: Config.BENCHMARK_ONYX,
    /* ... */
});
```